### PR TITLE
Rebranded Private Page and other changes

### PR DESCRIPTION
**Pull request by MasterK0927**
1) Rebranded the UI of Private aka Incognito Page
2) Changed the background image of new tab
3) Removed the "Scroll for Brave News" prompt from the New Tab page
4) Changed Brave -> Ping in the strings file responsible for private tab texts
5) Changed the default state of SideBar to NEVER i.e. hidden it
6) Removed the SideBar customisation settings from the settings/appearance tab
7) Changed the default state of Big Search bar to true
8) Set the default state of ShowSideBar Button to false

### CODE REVIEW

File: components/resources/brave_welcome_strings.grdp [hunk 1]
```diff
@@ -5,7 +5,7 @@
 </message>
 
 <message name="IDS_BRAVE_WELCOME_DESC" desc="A subtitle for screen on the first step of welcome page">
-  Get Brave protection on every link you click. Just set Brave as default and browse. Privately.
```
Comment (udaybansal19) [inline components/resources/brave_welcome_strings.grdp:8]: these changes were already merged right? why are they showing up again

File: components/img/newtab/backgrounds/dylan-malval_sea-min.webp
Comment (udaybansal19) [inline components/img/newtab/backgrounds/dylan-malval_sea-min.webp:1]: where have used the new file if this one is deleted

Comment (MasterK0927) [reply components/img/newtab/backgrounds/dylan-malval_sea-min.webp:1]: Deleted old file, and uploaded the new file with same name in the same folder

File: components/brave_welcome_ui/components/images/lion_logo.svg
Comment (udaybansal19) [inline components/brave_welcome_ui/components/images/lion_logo.svg:1]: these changes were also already merged. why are they showing up again.

Try creating a branch from this repo itself instead the fork and then raise a pr

Comment (udaybansal19) [reply components/brave_welcome_ui/components/images/lion_logo.svg:1]: ye ek icon delete karne se fix nahi hoga. tumhari pichli pr changes isme firse aa rahe hai. jo changes already merge ho gaye wo firse nahi ane chaiye na

File: components/brave_welcome_ui/assets/brave_logo_3d@2x.webp
Comment (udaybansal19) [inline components/brave_welcome_ui/assets/brave_logo_3d@2x.webp:1]: ye bhi toh last pr mein change kiya tha

File: components/resources/brave_welcome_strings.grdp [hunk 2]
```diff
@@ -49,31 +49,31 @@
 </message>
 
 <message name="IDS_BRAVE_WELCOME_SET_DEFAULT_BUTTON_LABEL" desc="A button that sets Brave browser as default">
-  Set Brave as default browser
+  Set Ping as default browser
```
Comment (udaybansal19) [inline components/resources/brave_welcome_strings.grdp:52]: ye sab bhi

File: components/brave_welcome_ui/stories/locale.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { provideStrings } from '../../../.storybook/locale'
 
 provideStrings({
   braveWelcomeTitle: 'Privacy. By Default.',
-  braveWelcomeDesc: 'Get Brave protection on every link you click. Just set Brave as default and browse. Privately.',
+  braveWelcomeDesc: 'Get Ping protection on every link you click. Just set Ping as default and browse. Privately.',
```
Comment (udaybansal19) [inline components/brave_welcome_ui/stories/locale.ts:10]: this as well

**Pull request closed without merging**